### PR TITLE
feat: update spec.emu

### PR DESCRIPTION
**Pull request by XGHeaven**

### CODE REVIEW

**Review by ljharb — CHANGES_REQUESTED** [review]
The spec file isn’t html, so the emu extension is intentional.

**Comment by XGHeaven** [comment]
@ljharb This is my first time writing a spec. I learned some proposals listed in https://github.com/tc39/proposals. I found most of the proposals using `.html` instead of `.emu`. And I'm also have not found any documents about the difference between .html and .emu.
So I changed the spec file extension to be the same as others.

**Comment by ljharb** [comment]
It's `.emu` in the template, and that was intentional. There's no documents because ecmarkup is a uniquely TC39 thing. I'd prefer it be changed back.

File: spec.html [hunk 1]
```diff
@@ -0,0 +1,652 @@
...
+        ImportCall[?Yield, ?Await]
+        <ins>BrandCheckCall[?Yield, ?Await]</ins>
+        CallExpression[?Yield, ?Await] Arguments[?Yield, ?Await]
+        CallExpression[?Yield, ?Await] `[` Expression[+In, ?Yield, ?Await] `]`
+        CallExpression[?Yield, ?Await] `.` IdentifierName
+        CallExpression[?Yield, ?Await] TemplateLiteral[?Yield, ?Await, +Tagged]
+        CallExpression[?Yield, ?Await] `.` PrivateIdentifier
+
+      <ins>BrandCheckCall[Yield, Await] :</ins>
+        `class` `.` `hasInstance` `(` AssignmentExpression[+In, ?Yield, ?Await] `)`
```
Comment (JLHwung) [inline spec.html:173]:
```suggestion
        `class` `.` `hasInstance` `(` AssignmentExpression[+In, ?Yield, ?Await] `,`? `)`
```
Should we allow trailing comma here? I would expect consistency between call expression and brand check call.

Comment (hax) [reply spec.html]: I believe we should allow that. I remember `import()` also have supported that.

**Comment by hax** [comment]
I'd like to merge this PR now. There are still some problems, let fix them in further PRs.

**Pull request merged**